osx/install_swift_binaries.sh

```diff
@@ -33,7 +33,17 @@ brew install wget > /dev/null || brew outdated wget > /dev/null || brew upgrade
 # See http://apple.stackexchange.com/questions/72226/installing-pkg-with-terminal
 # TODO: Since Xcode now includes the Swift compiler and Xcode is included in the macOS image provided
 # by Travis CI, we could add logic here that checks whether the Swift binaries are already available
-wget https://swift.org/builds/$SNAPSHOT_TYPE/xcode/$SWIFT_SNAPSHOT/$SWIFT_SNAPSHOT-osx.pkg
-sudo installer -pkg $SWIFT_SNAPSHOT-osx.pkg -target /
-export PATH=/Library/Developer/Toolchains/swift-latest.xctoolchain/usr/bin:"${PATH}"
-rm $SWIFT_SNAPSHOT-osx.pkg
+
+# Set the var to be the version of swift that is intalled.
+SWIFT_PREINSTALL="$(swift --version | awk '{print $5}' | sed 's/[)(]//g' )"
+SWIFT_PREINSTALL_MINIMAL="$(swift --version | awk '{print $5}' | sed 's/[)(]//g' | cut -b 7-11 )"
```

Comment:
@KyeMaloy97 I ran these commands on my macOS and got the following:
```
$ export SWIFT_PREINSTALL="$(swift --version | awk '{print $5}' | sed 's/[)(]//g' )"
$ echo $SWIFT_PREINSTALL
swiftlang-900.0.74.1

$ export SWIFT_PREINSTALL_MINIMAL="$(swift --version | awk '{print $5}' | sed 's/[)(]//g' | cut -b 7-11 )"
$ echo $SWIFT_PREINSTALL_MINIMAL
ang-9
```
Looking at the value assigned to the variables `SWIFT_PREINSTALL` and `SWIFT_PREINSTALL_MINIMAL`, it seems we need to do some tweaking to the code.

Also, note that the output from `swift --version` is a little different depending on the swift version:
```
$ swift --version
Apple Swift version 3.1 (swift-3.1.1-RELEASE)
Target: x86_64-apple-macosx10.9
```

```
 swift --version
Apple Swift version 4.0.3 (swiftlang-900.0.74.1 clang-900.0.39.2)
Target: x86_64-apple-macosx10.9
```
It looks like the part from that output that seems to be consistent across different versions of Swift is `Apple Swift version <version number>`.